Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -1011,29 +1015,28 @@ namespace OpenInfraPlatform {
 					for (int i = 1; i < numSegments; ++i) 
 					{
 						clothoidPoints.push_back(getPointOnCurve(clothoid, trim1Vec, trimmingPreference));
+						lengthNew += segmentLength;

```

Comment:
we somehow need to account for `lenrthNew` when calculating the points

Reply:
Resolved 2267315